socorro/unittest/external/boto/test_crashstorage.py

```diff
@@ -20,6 +26,8 @@
 from socorro.external.crashstorage_base import CrashIDNotFound
 import socorro.unittest.testbase
 
+TEMPDIR = tempfile.gettempdir()
```

Comment:
This pollutes. 
It leaves 
```
-rw-r--r--  1 peterbe  staff      29 Oct  7 12:29 936ce666-ff3b-4c7a-9674-367fe2120408.upload_file_minidump.TEMPORARY.dump
-rw-r--r--  1 peterbe  staff      36 Oct  7 12:29 936ce666-ff3b-4c7a-9674-367fe2120408.flash_dump.TEMPORARY.dump
-rw-r--r--  1 peterbe  staff      33 Oct  7 12:29 936ce666-ff3b-4c7a-9674-367fe2120408.city_dump.TEMPORARY.dump
``` 
in my temp directory when I run the tests. 
That's not great. 

I think what you should do is something like this:
...
Actually, to demonstrate I just wrote the code. I'll send this as a counter PR.